builder/packages_image_test.go

```diff
@@ -1,18 +1,62 @@
 package builder
 
 import (
+	"archive/tar"
+	"bytes"
+	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strings"
 	"testing"
 
 	"github.com/hpcloud/fissile/model"
-	"github.com/hpcloud/fissile/util"
 	"github.com/hpcloud/termui"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestCreatePackagesDockerBuildDir(t *testing.T) {
+func TestGenerateDockerfile(t *testing.T) {
+	assert := assert.New(t)
+
+	ui := termui.New(
+		os.Stdin,
+		ioutil.Discard,
+		nil,
+	)
+
+	workDir, err := os.Getwd()
+	assert.NoError(err)
+
+	compiledPackagesDir := filepath.Join(workDir, "../test-assets/tor-boshrelease-fake-compiled")
+	targetPath, err := ioutil.TempDir("", "fissile-test")
+	assert.NoError(err)
+	defer os.RemoveAll(targetPath)
+
+	packagesImageBuilder, err := NewPackagesImageBuilder("foo", compiledPackagesDir, targetPath, "3.14.15", ui)
+	assert.NoError(err)
+
+	baseImage := "scratch:latest"
```

Comment:
Kind of a useless variable. You could argue it's easy to see and easy to change, but you've lost that argument by using the hardcoded string again in the `assert.Equal` below. Use it in both places, or move it into `generateDockerfile()` IMHO